### PR TITLE
Improve transaction errors

### PR DESCRIPTION
**Pull request by RafaelTaranto**
Also bumped stardand version to match with l-s

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
Milestone: 7.3.x Crafty Chnemu

**Comment by RafaelTaranto** [comment]
[insert or update on table "cash_out_txs" violates foreign key constraint "cash_out_txs_customer_id_fkey"](https://trello.com/c/cuaYev8c/688-insert-or-update-on-table-cashouttxs-violates-foreign-key-constraint-cashouttxscustomeridfkey)

File: lib/trader.js [hunk 1]
```diff
@@ -191,9 +194,9 @@ function isNetworkError (err) {
 }
 
 Trader.prototype.stateChange = function stateChange (state, isIdle) {
-  this.state = {state: state, isIdle: isIdle}
+  this.state = { state: state, isIdle: isIdle }
```
Comment (joshmh) [inline lib/trader.js]:
```suggestion
  this.state = { state, isIdle }
```

File: lib/trader.js [hunk 2]
```diff
@@ -191,9 +194,9 @@ function isNetworkError (err) {
...
 
-  const rec = {state: state, isIdle: isIdle, uuid: uuid.v4()}
+  const rec = { state: state, isIdle: isIdle, uuid: uuid.v4() }
```
Comment (joshmh) [inline lib/trader.js]:
```suggestion
  const rec = { state, isIdle, uuid: uuid.v4() }
```

**Pull request merged**